plugin/adsk/scripts/mayaUSDRegisterStrings.mel

```diff
@@ -50,6 +50,7 @@ global proc mayaUSDRegisterStrings() {
     register("kMenuClear", "Clear");
     register("kMenuRevertToFile", "Revert to File");
     register("kMenuLayerEditor", "USD Layer Editor");
+    register("kContextMenuLayerEditor", "USD Layer Editor...");
```

Comment:
kMenuLayerEditor and kContextMenuLayerEditor look very similar. Just checking that it's not a mistake to have both.